packages/terra-list/docs/guides/SectionList.md

```diff
@@ -0,0 +1,119 @@
+# Terra List - Sections
+
+List sections are available in both static and collapsible varieties. A list section that is collapsed will be remove its children items from the DOM. This improves performance and accessiblilty, but may remove potentional state associated to a mounted list item. If your list item content has additional state, it needs to be stored externally and reapplied when thawed for the next render that shows its section open. Since this render occurs at the same time that a collapsible section state changes, for additional optimization, collapsed sections do not necessarily need to be provided with child items.
```

Comment:
nit: Change `that is collapsed will be remove its children items` 
to `that is collapsed will remove its children items`

Reply:
Updated.